ui/src/main/java/com/hyperwallet/android/ui/view/widget/DateParseException.java

```diff
@@ -0,0 +1,16 @@
+package com.hyperwallet.android.ui.view.widget;
+
+public class DateParseException extends Exception {
```

Comment:
don't make this public please we don't want to introduce new Exception handling routine for other UI's

Reply:
Done